src/mocks/mockComponent.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    BrsType,
+    ValueKind,
+    Callable,
+    BrsString,
+    StdlibArgument,
+    BrsInvalid,
+    RoAssociativeArray,
+} from "../brsTypes";
+import { MockNode } from "./MockNode";
+import { Interpreter } from "../interpreter";
+
+let mockComponent = new Callable("mockComponent", {
+    signature: {
+        args: [
+            new StdlibArgument("objToMock", ValueKind.String),
+            new StdlibArgument("mock", ValueKind.Object),
+        ],
+        returns: ValueKind.Dynamic,
```

Comment:
Probably there is no need to return anything here, can you try `ValueKind.void`?

Reply:
I am not able to get `ValueKind.Void` working. It is making me return _something_ and I don't understand the error messaging. Maybe we can look at it together